code_generator/GdkCodeGenerator/Templates/UnityGameObjectCommandHandlersGenerator.tt

```diff
@@ -50,10 +78,44 @@ namespace <#= qualifiedNamespace #>
             [InjectionCondition(InjectionCondition.RequireComponentWithAuthority)]
             public class CommandRequestHandler : IInjectable
             {
+                private Entity entity;
+                private readonly EntityManager entityManager;
+                private readonly ILogDispatcher logger;
+
                 public CommandRequestHandler(Entity entity, EntityManager entityManager, ILogDispatcher logger)
                 {
+                    this.entity = entity;
+                    this.entityManager = entityManager;
+                    this.logger = logger;
+                }
 
+<# foreach(var commandDetails in commandDetailsList) {
+           var commandRequestType = commandDetails.CommandName + ".ReceivedRequest";
+           var delegateList = commandDetails.CamelCaseName + "Delegates";
+#>
+                private readonly List<Action<<#= commandRequestType #>>> <#= delegateList #> = new List<Action<<#= commandRequestType #>>>();
+                public event Action<<#= commandRequestType #>> On<#= commandDetails.CommandName #>Request
```

Comment:
Just to confirm, this callback signature is subject to change when sending responses comes in, right?